src/PhpSessionPersistence.php

```diff
@@ -41,7 +41,7 @@
  */
 class PhpSessionPersistence implements SessionPersistenceInterface
 {
-    /** @var Cookie */
+    /** @var string|null */
     private $cookie;
```

Comment:
> $cookie is a Cookie value
> Q: should we just rename it to $sessionId?

This confused me a few times already. It's not a Cookie that is stored here but the session id.

Reply:
@xtreamwayz Yes, the name confused me at first as well, but even after using it for what it is (a session-id string or null), I actually had never noticed the misleading type-annotation, possibly a leftover of the initial implementation :-)